README.md

```diff
@@ -194,72 +137,7 @@ cd sandbox
 rails server
 ```
 
-Performance
------------
-
-You may notice that your Spree store runs slowly in development mode.  This is
-a side-effect of how Rails works in development mode which is to continuously reload
-your Ruby objects on each request.  The introduction of the asset pipeline in
-Rails 3.1 made default performance in development mode significantly worse. There
-are, however, a few tricks to speeding up performance in development mode.
-
-First, in your `config/environments/development.rb`:
-
-```ruby
-config.assets.debug = false
-```
-
-You can precompile your assets as follows:
-
-```shell
-RAILS_ENV=development bundle exec rake assets:precompile
-```
-
-If you want to remove precompiled assets (recommended before you commit to Git
-and push your changes) use the following rake task:
-
-```shell
-RAILS_ENV=development bundle exec rake assets:clean
-```
-
-Use Dedicated Spree Devise Authentication
------------------------------------------
-Add the following to your Gemfile
-
-```ruby
-gem 'spree_auth_devise', github: 'spree/spree_auth_devise'
-```
-
-Then run `bundle install`. Authentication will then work exactly as it did in
-previous versions of Spree.
-
-This line is automatically added by the `spree install` command.
-
-If you're installing this in a new Spree 1.2+ application, you'll need to install
-and run the migrations with
-
-```shell
-bundle exec rake spree_auth:install:migrations
-bundle exec rake db:migrate
-```
-
-change the following line in `config/initializers/spree.rb`
```

Comment:
@priyank-gupta this is only a note for legacy Spree versions (pre 2.x), using Spree with a custom setup (eg. external devise) should be covered in the guides